Tests/LibWeb/Screenshot/css-filter-drop-shadow.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html>
+<link rel="match" href="reference/css-filter-drop-shadow-ref.html" />
+<style>
+    div {
+        height: 20px;
+        width: 20px;
+        margin: 10px;
+        background-color: black;
+        color: blue;
+    }
+</style>
+<div style="filter: drop-shadow(5px 5px)"></div>
+<div style="filter: drop-shadow(5px 5px red)"></div>
+<div style="filter: drop-shadow(red 5px 5px)"></div>
+<div style="filter: drop-shadow(-5px -5px red)"></div>
+<div style="filter: drop-shadow(16px)"></div>
```

Comment:
While you're here, maybe also add blurred variants as well?

Reply:
Done :+1:

Reply:
I'll also take a crack at supporting `calc` in filters tomorrow